scripts/coords.js

```diff
@@ -0,0 +1,70 @@
+
+var _ = require('./constants.js');
+
+
+
+
+// -----------------------------+ private functions +--------------------------------
+
+function LatLon(lat, lon){
+	this.lat = lat;
+	this.lon = lon;
+}
+
+function toRad(n) {
+	return n * Math.PI / 180;
+};
+function toDeg(n) {
+	return n * 180 / Math.PI;
+};
+
+function segment(startLatLon, endLatLon) {
+	this.startLatLon = startLatLon;
+	this.endLatLon = endLatLon;
+}
+
+// -----------------------------+ exports +--------------------------------
+
+exports.convertLinksToDistance = function(numberOfLinks){
+	return numberOfLinks * _.linkDistance
+}
+// calculate the destination point given start point latitude / longitude (numeric degrees), bearing (numeric degrees) and distance (in m).
+// https://gist.github.com/mathiasbynens/354587
+exports.getDestinationLatLon = function(lat1, lon1, brng, dist) {
```

Comment:
mmm, it's so complex I don't want to touch it - it spits out the magic numbers though